SRT/Interfaces/Makefile

```diff
@@ -32,7 +32,6 @@ SUBSYSTEM = Interfaces
 MODULES = 	SRTActiveSurfaceInterface \
 		SRTAntennaInterface \
 		SRTReceiversInterface \
```

Comment:
The final \ should be taken out. Makefiles are very dumb and sometimes they will concatenate the next available line.

Reply:
This file won't actually impact anything since the SystemMake/Makefile is the one that matters, closing as resolved